scalafix-core/src/main/scala/scalafix/rewrite/NoExtendsApp.scala

```diff
@@ -14,18 +14,24 @@ case class NoExtendsApp(mirror: Mirror) extends SemanticRewrite(mirror) {
   override def rewrite(ctx: RewriteCtx): Patch = {
     def wrapBodyInMain(template: Template) =
       ctx
-        .templateBodyTokens(template)
-        .map { body =>
+        .templateBodyTokens(template) match {
+        case None =>
+          ctx.reporter.warn(
+            "Missing or empty body for object that extends scala.App. See http://dotty.epfl.ch/docs/reference/dropped/delayed-init.html for possible workarounds.",
+            template.pos
+          )
+          Patch.empty
```

Comment:
I agree it would make sense to accumulate messages, but why would they need to be attached to a `Patch`?